fe/fe-core/src/main/java/com/starrocks/load/InsertOverwriteJob.java

```diff
@@ -0,0 +1,104 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.load;
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.analysis.InsertStmt;
+
+import java.util.List;
+
+public class InsertOverwriteJob {
+    @SerializedName(value = "jobId")
+    private long jobId;
+
+    @SerializedName(value = "jobState")
+    private InsertOverwriteJobState jobState;
+
+    @SerializedName(value = "sourcePartitionNames")
+    private List<String> sourcePartitionNames;
```

Comment:
if you already have partitionid, why you need partition names?

Reply:
I modify it to use partition ids